posthog/rate_limit.py

```diff
@@ -222,6 +223,26 @@ def get_bucket_key(self, request):
         return ident
 
 
+class UserOrEmailRateThrottle(SimpleRateThrottle):
```

Comment:
i've only got a vague understanding these throttles...

if i'm a bad actor on a single IP stuffing email addresses into the login form will I get rate limited at some point?

Reply:
Nope. Only per email after this change.

Reply:
Should we have a fallback rate limit? 

It's relatively common for attackers to find sites with poor rate limits and use them to check email/common password combos to get around rate limits on the site they really want to attack...

Reply:
(can be a follow up since we want to test this really fixes the issue folk are experiening sooner rather than later)

Reply:
Login is protected differently (axes). This only applies to email validation / password reset requests.

Reply:
👍 someone could still stuff emails to find valid accounts but relatively low risk :)